src/components/controller/Contact/ContactProfileCard/index.js

```diff
@@ -0,0 +1,15 @@
+// import { connect } from 'react-redux';
```

Comment:
Why is this here, or why is it commented out?

Reply:
removed. thought I had needed it